src/lib.rs~

```diff
@@ -0,0 +1,449 @@
+/*  Copyright 2014 MaidSafe.net limited
```

Comment:
I think you have pushed temp files. Can you add a .gitignore and also remove these please.